Core/include/Acts/Vertexing/KalmanVertexUpdater.hpp

```diff
@@ -15,6 +15,14 @@ struct TrackAtVertex;
 
 namespace KalmanVertexUpdater {
 
+namespace detail {
```

Comment:
Probably I overlooked something, but at the first glance it seems to me that we could move this declaration to the source file? Not very important anyhow...

Reply:
Ah na we cannot of course...

Reply:
I think I'd have to copy it because I know have two source files that instantiate the template once each.

Reply:
Hmm I though about moving it `KalmanVertexUpdater.cpp`, where it is called... But not sure if that works...

Reply:
This is where it was located before, but then that compilation unit instantiates the template both with 3 and with 4, leading to the high memory usage of that compilation unit. The idea is to distribute this into two separate compilation units.

Reply:
Ah so it would instantiate it twice even if only the declaration and not the definition is there? (I only thought about moving the declaration). But yeah, probably not worth the time thinking to much about this.

Reply:
Ah maybe you're right and I can move the declaration only.